internal/jasmine_node_test/jasmine_runner.js

```diff
@@ -33,7 +33,7 @@ function main(args) {
       // deps such as "@npm//typescript" if executed may cause the test to
       // fail or have unexpected side-effects. "@npm//typescript" would
       // try to execute tsc, print its help, and process.exit(1)
-      .filter(f => /[^a-zA-Z0-9](spec|test)\.js$/i.test(f))
+      .filter(f => /[^a-zA-Z0-9](spec|test)((-|_|\.)[a-zA-Z0-9]|)\.js$/i.test(f))
```

Comment:
This doesn't seem to match `/[^a-zA-Z0-9](spec|test)((-|_|\.)[a-zA-Z0-9]|)\.js$/i.test('decrement.spec-foo.js')`.

Looks like the `|` in `[a-zA-Z0-9]|` should be `*` or `*?`?

Reply:
It should be `/[^a-zA-Z0-9](spec|test)((-|_|\.)[a-zA-Z0-9]+|)\.js`. The `|)` will allow still allow for `.spec.js` and `.test.js` file endings.

Reply:
👍